blockchain/blockchain.go

```diff
@@ -165,6 +165,7 @@ type BlockChain struct {
 	// State migration
 	prepareStateMigration bool
 	stopStateMigration    chan struct{}
+	readCnt               int
 	committedCnt          int
 	pendingCnt            int
 	progress              float64
```

Comment:
The variable names are not quite accurate as members in `Blockchain`.
This suggestion might not be the right place to write in this PR. However, what do you think about defining a new struct responsible for state migration?
```
types Blockchain struct {
...
  stateMigration stateMigration
...
}
```
```
type stateMigration struct {
	isPrepare  bool
	stopSignal chan struct{}
	Err        error

	migrationStats
}
```

Reply:
It's good idea. But I think modifying now is not good for this stage.